docusaurus/docs/create-a-plugin/extend-a-plugin/nested-plugins.mdx

```diff
@@ -1,7 +1,16 @@
 ---
 id: nested-plugins
-title: Nested plugins
-sidebar_position: 5
+title: Work with nested plugins
+sidebar_position: 12
+description: How to work with nested plugins, Grafana app plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - advanced
+  - apps 
+  - app plugins 
+  - nested
```

Comment:
The content in this page warrants follow up issue as its in no way obvious what we're suggesting for folks